frontend/src/app/passwordReset/VerifyAuthenticationApp/VerifyAuthenticationApp.tsx

```diff
@@ -0,0 +1,13 @@
+import { MfaVerify } from "../MfaVerify/MfaVerify";
+
+export const VerifyAuthenticationApp = () => (
+  <MfaVerify
+    hint={
+      <>
+        Enter a one-time security code from your authenticator application
+        (Google Authenticator or Okta)
+      </>
+    }
+    type="authentication app"
+  />
```

Comment:
Need to not display "Send another code" button from here on the `MfaVerify` component. And then do for auth pages in login and account setup. [Preview](https://www.chromatic.com/component?appId=60a556a7c807cc0039ec6786&name=App%2FPassword+reset%2FVerify%3A+Authentication+app&mode=interactive&buildNumber=54)

Reply:
cc @cdevent